src/proto/mgmt/acl.proto

```diff
@@ -14,9 +14,9 @@ option go_package = "github.com/daos-stack/daos/src/control/common/proto/mgmt";
 // Response to ACL-related requests includes the command status and current ACL
 message ACLResp {
 	int32 status = 1; // DAOS error code
-	repeated string ACL = 2; // List of ACEs in short string format
-	string ownerUser = 3; // Name of user that owns the resource
-	string ownerGroup = 4; // Name of group that owns the resource
+	repeated string entries = 2; // List of ACEs in short string format
+	string owner_user = 3; // Name of user that owns the resource
+	string owner_group = 4; // Name of group that owns the resource
```

Comment:
It might be overengineering, but just a thought: You could break these out into a standalone `ACL` message. For this specific `ACLResp` message, we're only expecting one ACL, but I could imagine a future use case where we're wanting to get multiple ACLs in a RPC response (or to send them in a request), and making this change now while you're doing other breaking changes might be better.

Reply:
That's a good idea. I'll make that change.